src/main/java/com/grapeup/webapp/testWebApp/exceptions/SessionException.java

```diff
@@ -0,0 +1,9 @@
+package com.grapeup.webapp.testWebApp.exceptions;
+
+public class SessionException extends RuntimeException {
```

Comment:
Another thing which you could try for such simple use case is just annotate exception with `@ResponseStatus`, Spring will serialize exception to response with given HTTP status.